src/main/java/teammates/ui/webapi/action/UpdateCourseAction.java

```diff
@@ -58,7 +62,6 @@ public ActionResult execute() {
             return new JsonResult(edee.getMessage(), HttpStatus.SC_NOT_FOUND);
         }
 
-        return new JsonResult("Updated course [" + courseId + "] details: Name: " + courseName
-                + ", Time zone: " + courseTimeZone, HttpStatus.SC_OK);
+        return new JsonResult(new CourseData(logic.getCourse(courseId)));
```

Comment:
`updateCourseCascade` method will return the updated entity. There is no need to query it again here.